app/src/main/java/com/example/concordia_campus_guide/Database/Daos/BuildingDao.java

```diff
@@ -0,0 +1,32 @@
+package com.example.concordia_campus_guide.Database.Daos;
+
+import androidx.room.Dao;
+import androidx.room.Delete;
+import androidx.room.Insert;
+import androidx.room.Query;
+import androidx.room.Transaction;
+
+import com.example.concordia_campus_guide.Models.Building;
+import com.example.concordia_campus_guide.Models.Relations.BuildingWithFloors;
+
+import java.util.List;
+
+@Dao
+public interface BuildingDao {
+
+    @Query("SELECT * FROM buildings")
```

Comment:
can't we retrieve 1 building at a time? is it not a necessary feature?

Reply:
we will be able to do so, we will have to write this as it goes.